### PR TITLE
Display Comments pop up with selected item's details

### LINKED ISSUE #14: [4pt]  -Display Comments pop up with selected item's details - Student B
Create a comments popup only with the top section (displaying details of the selected item)

Screenshot 2021-06-27 at 22 10 45

When the popup loads, the webapp retrieves data from the selected API and shows details about the selected item.

Add event to button prepared by your teammate that will open your popup. You need to communicate with your team member to make sure that you do not block each other (most likely the easiest solution is to merge the list of items feature to the dev branch before you try to add that event).

### PR DESCRIPTION
**Pull request by Camilovelag**
# Good day @sntgopl ! 🌤️

This pull request is intended to display a popup for comments for a selected pokemon.

## 🛠️ Changes made

✔️ Create a popup window dynamically.
✔️ Fetch data from pokeAPI.

We're expecting feedback.

Regards!  👏👏👏

done

### CODE REVIEW

**Comment by sntgopl** [comment]
# Good day, @Camilovelag!

Great job you`ve done there with the popUp and the information 'fetch' from the API with the Pokemon. Great Caterpillar!

## Changes required
I believe there are some things we should work on regarding the popUp:
- Let`s try to give the window some styling and position like the one in the template. 
- Try to add a section 'Comments' and a form with a text-area to write the comments.

Let me know if there's anything I could help you with. The functionality looks great, and I see you've already reached the API for displaying the picture.

**Comment by sntgopl** [comment]
# Good day, @Camilovelag!
Great job you`ve done there with the popUp and the information 'fetch' from the API with the Pokemon. Great Caterpillar!

## Changes required
I believe there are some things we should work on regarding the popUp:
- Let`s try to give the window some styling and position like the one in the template.
- Try to add a section 'Comments' and a form with a text-area to write the comments.
- Let's check about that linter error later, it must be something with the Jest installation.

Let me know if there's anything I could help you with. The functionality looks great, and I see you've already reached the API for displaying the picture.

File: src/index.html [hunk 1]
```diff
@@ -11,8 +11,9 @@
     <section id="homePage">
 
     </section>
+    <button id="tmp-popup-open" type="button">Hey, I'm a popup</button>
+    <!-- change secction name to popup - standard notation -->
     <section id="popUp">
```
Comment (sntgopl) [inline src/index.html]: Maybe the form and the div for the popup can be here and then append dynamically the fetch from the API

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+html * {
+  margin: 0;
+  padding: 0;
+}
+
+#popUp {
```
Comment (sntgopl) [inline src/style.css:7]: Try adding some styling to the window popUp. Maybe based on the template.

**Review by sntgopl — CHANGES_REQUESTED** [review]
Good job so far, Camilo.

Keep on!

File: src/index.html (hunk 1 above)
Comment (Camilovelag) [reply src/index.html]: Actually, the popup is generated dynamically using `renderPopup.js`  via `fetchPopup.js`, contained in the `popUp` section. The `tmp-popup-open` button is a temporary button created to call the popup window before merging with the homepage feature.

File: src/style.css (hunk 2 above)
Comment (Camilovelag) [reply src/style.css:7]: Done! 😄

File: src/index.html (hunk 1 above)
Comment (Camilovelag) [reply src/index.html]: I mean, the popup is inserted into the `popUP` section 😄

**Comment by sntgopl** [comment]
# Great job, Camilo. You did it great. Let`s merge it already!

**Review by sntgopl — APPROVED** [review]
Approved changes. Let's try to merge it.

**Pull request merged**